app/services/publish_finder.rb

```diff
@@ -24,25 +23,6 @@ def send_to_publishing_api
     Services.publishing_api.publish(content_id)
   end
 
-  def send_to_rummager
-    index = Whitehall::SearchIndex.for(:government)
-    index.add(present_for_rummager)
-  end
-
-  def present_for_rummager
```

Comment:
This method is still being called [here](https://github.com/alphagov/whitehall/blob/d5da9476e64ebba0068cc1e045a2d1d562515e92/lib/publish_static_pages.rb#L5). Deleting lines 3 and 5 from that class should stop it.